webdriver-helpers/configuration-helper.js

```diff
@@ -9,7 +9,7 @@ exports.getAppConfig = (appName) => {
             config.amplifyId = VANILLA_AMPLIFY_ID;
             config.port = '3001';
             break;
-        case 'nextjs-app':
+        case 'nextjs-app-v13':
```

Comment:
At what point will we add a case for v14?

Reply:
In the follow up tickets which are ready to be worked on.